back/.rubocop.yml

```diff
@@ -143,7 +143,6 @@ RSpec/ExampleLength:
 
 Metrics/MethodLength:
   CountAsOne: ['array', 'hash', 'heredoc']
-  AllowedMethods: ['included', 'prepended', 'extended']
```

Comment:
I kept getting warnings in the VS code Rubocop integration that AllowedMethods was not supported by Metrics/MethodLength. Removing this line didn't introduce any offenses.